repository/c.json

```diff
@@ -4269,6 +4269,16 @@
 				}
 			]
 		},
+		{
+			"name": "CSharpier",
+			"details": "https://github.com/mkonstapel/sublime-text-csharpier-plugin",
```

Comment:
You might want to add labels like [formatting](https://packagecontrol.io/browse/labels/formatting)

Reply:
Done!